lib/Composer.php

```diff
@@ -216,7 +216,7 @@ protected static function getPhpArguments()
             $arguments = $phpFinder->findArguments();
         }
 
-        if ($env = getenv('COMPOSER_ORIGINAL_INIS')) {
+        if ($env = $_SERVER['COMPOSER_ORIGINAL_INIS']) {
```

Comment:
```suggestion
        if ($env = $_SERVER['COMPOSER_ORIGINAL_INIS'] ?? null) {
```

Reply:
Have fixed it with `!empty($_SERVER['SYMFONY_ASSETS_INSTALL'])` - is also easier to read.